### DISCUSSION TITLE
License is for Chrome but extension build is for Firefox

### DISCUSSION THREAD

**Discussion by nomadrt** [aclap-dev/video-downloadhelper#62] [License & Purchase requests]
### What's your issue?

I want to move the license to firefox, I reset through the email but still encountered the message "License is for Chrome but extension build is for Firefox". I tried to move the license to firefox to download from youtube. Let me know what I need to do?

### The 10 first digits (only 10) of your license number or email address

3f99ae117f

### I confirm that I didn't not paste my full license number

- [x] I confirm

**Comment by paulrouget** [comment]
Send a message here: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem (License Problem section).
